catalog/CatalogDatabaseCache.cpp

```diff
@@ -61,15 +61,17 @@ void CatalogDatabaseCache::update(const serialization::CatalogDatabase &proto) {
 
   SpinSharedMutexExclusiveLock<false> lock(relations_mutex_);
   for (int i = 0; i < proto.relations_size(); ++i) {
-    auto relation_schema = make_unique<const CatalogRelationSchema>(proto.relations(i));
-    const relation_id rel_id = relation_schema->getID();
+    const serialization::CatalogRelationSchema &proto_relation = proto.relations(i);
+    const relation_id rel_id = proto_relation.relation_id();
```

Comment:
@pateljm Do you mean `relation_id` in `proto_relation.relation_id()`?

If so, it is the API generated by protobuf. Thanks!